GMX-Bots/Unusual_PnL/src/agent.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { ethers, Finding, FindingSeverity, FindingType, HandleTransaction, TransactionEvent } from "forta-agent";
+import { provideBotHandler } from "./agent";
+import { TestTransactionEvent, createAddress } from "forta-agent-tools/lib/tests.utils";
+import { encodeParameter } from "forta-agent-tools/lib/utils";
+
+import NetworkManager from "./network";
+import BigNumber from "bignumber.js";
+BigNumber.set({ DECIMAL_PLACES: 18 });
+const testCreateFinding = (positionSize: BigNumber, realisedPnl: BigNumber, key: string, contract: string): Finding =>
+  Finding.fromObject({
+    name: "Unusual PnL",
+    description: "ClosePosition Event with a high pnl emitted from GMX's Vault contract",
+    alertId: "GMX-6",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    protocol: "GMX",
+    metadata: {
+      "Realised PnL": realisedPnl.toFixed(2),
+      "Position size": positionSize.toFixed(2),
+      GMX: contract,
+      "Position key": key,
+    },
+  });
+
+const TEST_GMX_VAULT = createAddress("0x256H");
```

Comment:
H is not a valid character, everything is going to work if you use invalid characters but please avoid to do it.